### PR TITLE
Wait for machineConfigPool after applying SSL cert

### PR DESCRIPTION
**Pull request by petr-balogh**
Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/ssl_certs.py [hunk 1]
```diff
@@ -266,6 +270,7 @@ def configure_custom_ingress_cert():
         '--type=merge -p \'{"spec":{"defaultCertificate": {"name": "ocs-cert"}}}\''
     )
     exec_cmd(cmd)
+    wait_for_machineconfigpool_status("all", timeout=1800)
```
Comment (prsurve) [inline ocs_ci/utility/ssl_certs.py:273]: we need to sleep here for 10 sec and then we can start for wait_for_machineconfigpool_status

Comment (clacroix12) [reply ocs_ci/utility/ssl_certs.py:273]: @prsurve why is the sleep necessary? / If it's always necessary should the `wait_for_machineconfigpool_status` function handle that wait time for us?

Comment (prsurve) [reply ocs_ci/utility/ssl_certs.py:273]: It's because mcp takes few sec to update with new config

Comment (petr-balogh) [reply ocs_ci/utility/ssl_certs.py:273]: I added it directly to the function in next commit and removed from all other places as I noticed we are waiting 60 seconds before calling this function on all the places.

**Review by vavuthu — APPROVED** [review]

**Review by jimdevops19 — APPROVED** [review]

**Pull request merged**